.appveyor.yml

```diff
@@ -0,0 +1,82 @@
+image: Visual Studio 2015
+version: '{branch}.{build}'
+
+cache:
+  - c:\build-cache -> .appveyor.yml
+  - c:\build-cache\sdk -> .appveyor.yml
+
+environment:
+  PHP_BUILD_CACHE_BASE_DIR: c:\build-cache
+  PHP_BUILD_OBJ_DIR: c:\obj
+  PHP_BUILD_CACHE_SDK_DIR: c:\build-cache\sdk
+  PHP_BUILD_SDK_BRANCH: php-sdk-2.1.1
+  SDK_REMOTE: https://github.com/OSTC/php-sdk-binary-tools.git
+  SDK_BRANCH: php-sdk-2.1.1
+
+  matrix:
+    - PHP_REL: 7.2
+      ARCHITECTURE: x64
+      ZTS_STATE: enable
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      PHP_BUILD_CRT: vc15
+    - PHP_REL: 7.1
+      ARCHITECTURE: x64
+      ZTS_STATE: enable
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      PHP_BUILD_CRT: vc14
+    - PHP_REL: 7.0
+      ARCHITECTURE: x64
+      ZTS_STATE: enable
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      PHP_BUILD_CRT: vc14
+    - PHP_REL: 7.2
+      ARCHITECTURE: x64
+      ZTS_STATE: disable
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      PHP_BUILD_CRT: vc15
+    - PHP_REL: 7.1
+      ARCHITECTURE: x64
+      ZTS_STATE: disable
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      PHP_BUILD_CRT: vc14
+    - PHP_REL: 7.0
+      ARCHITECTURE: x64
+      ZTS_STATE: disable
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      PHP_BUILD_CRT: vc14
+    - PHP_REL: 7.2
+      ARCHITECTURE: x86
+      ZTS_STATE: enable
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      PHP_BUILD_CRT: vc15
+    - PHP_REL: 7.1
+      ARCHITECTURE: x86
+      ZTS_STATE: enable
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      PHP_BUILD_CRT: vc14
+    - PHP_REL: 7.0
+      ARCHITECTURE: x86
+      ZTS_STATE: enable
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      PHP_BUILD_CRT: vc14
+    - PHP_REL: 7.2
+      ARCHITECTURE: x86
+      ZTS_STATE: disable
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      PHP_BUILD_CRT: vc15
+    - PHP_REL: 7.1
+      ARCHITECTURE: x86
+      ZTS_STATE: disable
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      PHP_BUILD_CRT: vc14
+    - PHP_REL: 7.0
+      ARCHITECTURE: x86
+      ZTS_STATE: disable
+      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      PHP_BUILD_CRT: vc14
```

Comment:
No real reason, except that we'd be creating binaries for outdated PHP versions, and building binaries takes a looong time on AppVeyor already.

Reply:
I think it's worth adding that if we're to rely on this instead of manually creating test builds for PHP 5.5 and 5.6 before releasing.

If build times are very long, I'd also be amenable to disabling Appveyor for PRs and simply having it monitor branches on the mongodb repo.

Reply:
I tried that, but no luck:
https://ci.appveyor.com/project/derickr/mongo-php-driver/build/master.18/job/yc9jeqbrawjkuh97#L51

Reply:
The new SDK only supports 2015 and 2017: https://github.com/Microsoft/php-sdk-binary-tools#requirements